lib/bootscale/entry.rb

```diff
@@ -13,6 +13,7 @@ def initialize(path)
     end
 
     def features
+      return [] unless @path.start_with?('/'.freeze)
```

Comment:
I see 3 options:
 - do not cache relative -> correctness issues when having multiple similar paths
 - cache as absolute -> correctness issues when using Dir.chdir (idk .. if that's really a big issue ...)
 - cache as `:relative` and then ignore them when resolving

the last one would be as correct as possible ... but I don't think Dir.chdir is a big issue / I hope ppl do that weirdness on purpose ...